packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -65,9 +65,8 @@ exports.questions = {
 	conservationAreaUpload: new MultiFileUploadQuestion({
 		title: 'Conservation area map and guidance',
 		question: 'Upload conservation map and guidance',
-		description:
-			'<a href="https://magic.defra.gov.uk/magicmap.aspx" target="_blank" class="govuk-link">Link to Magic Maps (opens in new tab)</a>',
-		fieldName: 'conservation-area-upload'
+		fieldName: 'conservation-area-upload',
+		validators: [new RequiredValidator('You must add your documents')]
```

Comment:
for multifile validation, this line should now read:
`validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]`